vcloud-director/src/main/java/brooklyn/networking/vclouddirector/NatService.java

```
@@ -549,37 +549,17 @@ private boolean includesPublicIp(final String publicIp, List<SubnetParticipation
         return false;
     }
     
-    private void checkPublicIps(Iterable<String> publicIps, List<SubnetParticipationType> subnetParticipations) {
+    private boolean checkPublicIps(Iterable<String> publicIps, List<SubnetParticipationType> subnetParticipations) {
```

The naming convention of `checkPublicIps` follows the guava contention. The semantics elsewhere are that it will throw an exception if the check fails, rather than returning a boolean.